core/arch/arm/pta/device.c

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (C) 2019, Linaro Limited
+ */
+
+/*
+ * This pseudo TA is used by Linux TEE driver to fetch pseudo/early TA's UUIDs
+ * which can act as TEE bus devices.
+ */
+
+#include <kernel/pseudo_ta.h>
+#include <kernel/tee_ta_manager.h>
+#include <tee/uuid.h>
+#include <pta_device.h>
+#include <string.h>
+#include <user_ta_header.h>
+
+#define PTA_NAME "device.pta"
+
+static uint32_t num_devices;
```

Comment:
I'd prefer if we didn't have this global variable.

Reply:
ok